website/static/js/tests/paginator.test.js

```diff
@@ -109,8 +110,8 @@ describe('Paginator', () => {
 
         it('more than 7 pages, currentPage more than numbersOfPages - 5, one ellipse at the beginning',
             () => {
```

Comment:
Same as above -- please change this comment to reflect the changes you made for the test.